Assets/Scripts/Inventory.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Inventory : MonoBehaviour
+{
+    [SerializeField]
+    private List<Item> items = new List<Item>();
+
+    public void Add(Item item)
```

Comment:
Just as a hint :) Methods like these can be simplified to:
```c#
public void Add(Item item) => items.Add(item);
```

Reply:
Thanks, I've included that.